src/item.cpp

```diff
@@ -3983,10 +3966,10 @@ long item::ammo_required() const
     return 0;
 }
 
-bool item::ammo_consume( int qty, const tripoint& pos ) {
+long item::ammo_consume( long qty, const tripoint *pos ) {
```

Comment:
> Nothing to stop someone supplying tripoint_min instead of refactoring to be able to supply the correct position.

In the same way nothing stops anyone from supplying it right now.

> As it gradually becomes a replacement for manipulating charges directly the situation often recurs.

I can't think of an example where it would ever not be bad design not to supply the position.